app/default/.elasticsearch-http-requests/request.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/sh
```

Comment:
@212223 why this directory move from `app` to `app/default` here and in lots of places? 🤔

Reply:
It is good to have a web application served by `nginx` from a sub-directory `app/default` instead of a parent directory `app` because this allows to switch between applications instantaneously.

For example instead of having your application inside the `app/default` dir you may have it in `app/bar` dir and a [symlink](https://github.com/212223/painless-car-rental/blob/c480a94a6f718d54157444105dbd045c9937e60e/app/default) `app/default -> app/bar` that points to the dir of the application you want to serve. [Demo commit](https://github.com/212223/painless-car-rental/commit/c480a94a6f718d54157444105dbd045c9937e60e).

Then if you want to change the application you serve you don't need to reconfigure nginx nor restarting any of the services is required but only changing the symlink target, for example:

```
cd app
unlink default
ln -sr foo default
```
like in the [Demo commit](https://github.com/212223/painless-car-rental/commit/bee580802f5cd53adb758f086b1aaa9000e4b987)

and you are ready to serve the new application. 
Example use cases:
- Live update of an application
- Rollback to previous version of an app in case of troubles.
- Presenting different applications or the same in different versions during a time constrained presentation.
- Fallback to an application that helps debugging problems.
- Creating a new version of an app just by copying it to a new dir and symlink to it.

Currently the web app is in the `app/default` dir. Would you like me to:
1. Leave it as it is
2. Move it to a different dir and create a symlink `default` that points to that dir
3. Move that application to `app` dir from `app/default` and reconfigure docker-compose?